frontends/web/src/routes/settings/components/appearance/activeCurrenciesDropdownSetting.tsx

```diff
@@ -25,13 +26,20 @@ const ActiveCurrenciesDropdownSetting = () => {
   const { t, i18n } = useTranslation();
   const { activeCurrencies, defaultCurrency } = useContext(RatesContext);
   const { formattedCurrencies } = useLocalizedFormattedCurrencies(i18n.language);
+  const [isMobileDropdownOpen, setIsMobileDropdownOpen] = useState(false);
+  const isMobile = useMediaQuery('(max-width: 768px)');
+
   return (
     <SettingsItem
+      disabled={!isMobile}
       collapseOnSmall
       settingName={t('newSettings.appearance.activeCurrencies.title')}
       secondaryText={t('newSettings.appearance.activeCurrencies.description')}
+      onClick={() => setIsMobileDropdownOpen(true)}
```

Comment:
What do you think about conditionally adding this callback in case isMobielDropdownOpen is false.

Then SettingsItem should not render as button in case the dropdown is open.....